### PR TITLE
chore: replace twitter links to RISC Zero blog.

### PR DESCRIPTION
**Pull request by nuke-web3**
Twitter requires an account to read things, our blog does not :grin: 

`lychee` passes for all links :grin:

### CODE REVIEW

**Comment by nuke-web3** [comment]
CI should not run for tests when I touch .md files only :see_no_evil:

**Comment by nuke-web3** [comment]
draft to resolve what the right link to use is now:

> Taking a look the Foundry template now to see if it needs updates. Near the top, we link to this article, which I imagine is causing some confusion. In particular, it describes the relay based architecture, which has now been removed. It's always hard to keep these things up to date. Do y'all have thought on what we should do here? Maybe we start with simply removing the link?

**Comment by pdg744** [comment]
Thanks for updating this! The twitter thread came out before the blog post and apparently forgot to update the links when the blog came out. 

As I commented on Slack, I'm inclined toward making some minor revisions to the blogpost to remove the now-stale-descriptions-of-architecture.

**Comment by pdg744** [comment]
Process comment for future PRs: We generally only push docs improvements to `api/` and the current version of docs. In the interest of keeping PRs streamlined and minimizing maintenance work, let's generally leave the historical docs as-they-are, unless there's a compelling reason to go back and change the older content.

**Comment by pdg744** [comment]
> draft to resolve what the right link to use is now

I think we can go ahead and deprecate the twitter links in favor of a pointer to the blogpost. Thanks for helping do due diligence on that question :)

File: examples/chess/README.md [hunk 1]
```diff
@@ -21,10 +21,10 @@ Congratulations! You just constructed a zero-knowledge proof that the position d
 By using this demo as part of a [Bonsai application], you could build an app where on-chain payment depends on off-chain activity.
 For example, you might issue an on-chain award for the first person who can find a mate-in-one.
 
-To learn more about this use case, check out our Twitter thread about using Bonsai as a [zk coprocessor].
+To learn more about this use case, check out our blog about using Bonsai as a [zk coprocessor].
 
 [Bonsai application]: https://dev.bonsai.xyz
-[zk coprocessor]: https://twitter.com/RiscZero/status/1677316664772132864
+[zk coprocessors]: https://www.risczero.com/blog/a-guide-to-zk-coprocessors-for-scalability
```
Comment (pdg744) [inline examples/chess/README.md:27]: I think the additional `s` was unintentional, ya?

```suggestion
[zk coprocessor]: https://www.risczero.com/blog/a-guide-to-zk-coprocessors-for-scalability
```

**Review by pdg744 — APPROVED** [review]
Thanks Nuke!

File: examples/chess/README.md [hunk 2]
```diff
@@ -52,24 +49,21 @@ The [guest code] checks that applying the move to the initial board state is leg
...
+[host code]: methods/guest/src/main.rs
 [host program]: https://dev.risczero.com/terminology#host-program
+[host]: https://dev.risczero.com/terminology#host
+[journal]: https://dev.risczero.com/terminology#journal
 [proves the execution]: https://dev.risczero.com/terminology#prove
 [receipt]: https://dev.risczero.com/terminology#receipt
-[verify]: https://dev.risczero.com/terminology#verify
-[journal]: https://dev.risczero.com/terminology#journal
-[examples guide]: https://dev.risczero.com/api/zkvm/examples/#running-the-examples
-[here]: https://github.com/risc0/risc0/blob/main/examples/chess/src/main.rs#L29
```
Comment (nuke-web3) [inline examples/chess/README.md:66]: just for this file as an example:

- moved all links to the bottom & sorted alphabetically (VSCode has a nice select lines -> ctrl+shift+p -> sort ascending flow to help)
- removed all hard coded github self-referencing links to this repo to relative links
- de-duplicated link referenced
- pass lyche for this file

If this looks good, we can do do this kinda thing as a chore systematically throughout for .md files at least

File: examples/chess/README.md (hunk 1 above)
Comment (nuke-web3) [reply examples/chess/README.md]: indeed - I cleaned up this file's links & checked against lychee to be sure everything should be working now :grin:

**Pull request merged**